config/win/BUILD.gn

```diff
@@ -376,6 +387,20 @@ config("common_linker_setup") {
     "/DYNAMICBASE",
   ]
 
+  # ASLR makes debugging with windbg difficult because Chrome.exe and
```

Comment:
How did you find this information about chrome? just wondering where it came from

Reply:
That is Google comment and it was present in older version of that file. I believe that is from m62.